.drone.yml

```diff
@@ -4,40 +4,41 @@ workspace:
 
 services:
   web:
-    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:7.3-ci}
+    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:8.0-ci}
```

Comment:
Use a parameter for the php version from the matrix (after is fixed) instead of passing the actual value.